### PR TITLE
Feat/front metadata request

### PR DESCRIPTION
**Pull request by lucasbordeau**
- Added a new Apollo client on `/metadata`, that gets its auth from token pair
- Set this client in a Context
- Fetch metadata in an Effect component on page first load
- Changed FilterOperand to ViewFilterOperand to avoid type errors with generated types from graphql-codegen (could be changed back after we fix schema introspection bugs)

### CODE REVIEW

File: front/src/metadata/states/apolloClientMetadataState.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { ApolloClient, NormalizedCacheObject } from '@apollo/client';
+import { atom } from 'recoil';
+
+export const apolloClientMetadataState =
+  atom<ApolloClient<NormalizedCacheObject> | null>({
+    key: 'apolloClientMetadataState',
```
Comment (charlesBochet) [inline front/src/metadata/states/apolloClientMetadataState.ts:6]: is this storing the cache? I'm unsure why we need it?

**Review by charlesBochet — COMMENTED** [review]
Nice piece of work!

File: front/src/metadata/states/apolloClientMetadataState.ts (hunk 1 above)
Comment (lucasbordeau) [reply front/src/metadata/states/apolloClientMetadataState.ts:6]: This is useless now, it was part of a test

File: front/src/generated-metadata/fragment-masking.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
+import { ResultOf, DocumentTypeDecoration, TypedDocumentNode } from '@graphql-typed-document-node/core';
```
[the remark this answers is not in the document]
Comment (lucasbordeau) [reply front/src/generated-metadata/fragment-masking.ts:1]: Introduced by the new version of graphql-codegen : https://the-guild.dev/graphql/codegen/docs/guides/react-vue

**Review by charlesBochet — APPROVED** [review]
LGTM! A good step forward!

**Pull request merged**